src/common/components/AddToMetaMaskButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+declare global {
+  interface Window {
+    ethereum: any;
+  }
+}
```

Comment:
What would you think of doing:

```suggestion
import { MetaMaskInpageProvider } from "@metamask/providers";

declare global {
  interface Window {
    ethereum?: MetaMaskInpageProvider;
  }
}
```

The package originates from the official [MetaMask provides repo](https://github.com/MetaMask/providers). This would avoid the `any` and provide us with some better type information.

Reply:
Fixed in https://github.com/iota-wiki/iota-wiki/pull/919/commits/df32d9dd4e4446decc6aeba1d63804a57168bdd3